pages/linux/distrobox-host-exec.md

```diff
@@ -0,0 +1,13 @@
+# distrobox-host-exec
+
+> Execute a command on host while inside a distrobox container.
+> Subcommand of `distrobox`. More about distrobox: `tldr distrobox`.
+> More information: <https://github.com/89luca89/distrobox/blob/main/docs/usage/distrobox-host-exec.md>.
+
+- Execute command on the host system, while inside of a container:
+
+`distrobox-host-exec {{command}}`
```

Comment:
```suggestion
`distrobox-host-exec "{{command}}"`
```

Optional fix: command placeholder can be quoted if it can consist of multiple terms.